waspc/e2e-test/GoldenTest.hs

```diff
@@ -83,3 +99,24 @@ runGoldenTest goldenTest = do
     writeFileManifest baseAbsFp filePaths manifestAbsFp = do
       let sortedRelativeFilePaths = unlines . sort . map (makeRelative baseAbsFp) $ filePaths
       writeFile manifestAbsFp sortedRelativeFilePaths
+
+    -- This function normalizes all package.json files for comparison as `npm install` can overwrite
+    -- them when creating/updating package-lock.json. Different versions of npm may produce different
+    -- (but semantically equivalent) package.json files, thus triggering false positives.
+    -- Ref: https://github.com/wasp-lang/wasp/issues/482
```

Comment:
```suggestion
    -- While Wasp deterministically produces package.json files in the generated code, - 
    -- later calls to `npm install` can reformat them (e.g. it sorts dependencies).
    -- Also, Different versions of npm may produce different (but semantically equivalent) 
    -- package.json files.
    -- All of this can result in false negatives in e2e tests.
    -- Ref: https://github.com/wasp-lang/wasp/issues/482
```

Reply:
Hm now I wonder, is a failed test a positive or negative :D?
I thought failed test is negative, but now I realized it might not be, it might be considered a positive?

Reply:
Haha yeah its tricky to think if it false positive or negative... I guess it depends on what perspective one takes. I will change to "can result in being flagged as different when it should not be" or something.